### ISSUE TITLE
Training Settings

### ISSUE THREAD

**Issue by Yeez-lee** [sail-sg/MDT#47]
Hi, I am curious about your MDTv2_s2 settings. What is your lr_anneal_steps? Should I set it to 0? And how can I control the training steps? For example, how can I set the overall training steps to 400k? Thank you!

**Comment by gasvn** [comment]
We keep the same learning rate in our setting. And we just kill the running program when it reaches the required training step.

**Comment by Yeez-lee** [comment]
> We keep the same learning rate in our setting. And we just kill the running program when it reaches the required training step.

Thank you! One more question is about cfg_scale. Do you always keep 3.8 for S, B, and XL models during inference?

**Comment by gasvn** [comment]
Yes, the value is obtained with MDTv1, and I keep using 3.8 for all models of MDTv2
